docs/azure/sdk/snippets/authentication/credential-chains/dac-logs.txt

```diff
@@ -0,0 +1,11 @@
+DefaultAzureCredential.GetToken invoked. Scopes: [ https://api.loganalytics.io//.default ] ParentRequestId: d7ef15d1-50f8-451d-afeb-6b06297a3342
```

Comment:
I'm concerned about the potential of security concerns with this, are any of these GUIDs, URLs, or details something to worry about?

Reply:
There's nothing of concern in these logs. By default, the logging of sensitive data is disabled. See https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity#logging.